osbuild/main_cli.py

```diff
@@ -211,6 +211,5 @@ def osbuild_cli() -> int:
             return 0 if r["success"] else 1
 
     except KeyboardInterrupt:
-        print()
```

Comment:
What's the reasoning behind removing this instance of `print()` and still keeping it e.g. on line 208? :thinking: